news_items/034_matching_element/info.json

```diff
@@ -0,0 +1,6 @@
+{
+    "uuid": "bed17803-fce6-4732-96e1-4b91e30a907b",
+    "title": "New pl-matching element",
+    "author": "<a href=\"https://github.com/nealterrell\">Neal Terrell</a>",
```

Comment:
We generally have someone other than the PR author listed as the news item author, to avoid having it look like someone is praising themselves in the news item 😄. Maybe @jonatanschroeder could be the news item author?

Reply:
Do you want a joint news item that includes the other elements? In that case it may need to be you...